.github/workflows/style-check.yml

```diff
@@ -66,9 +66,12 @@ jobs:
           java-version: '17'
           distribution: 'temurin'
       - name: Google Java Format
-        uses: axel-op/googlejavaformat-action@fe78db8a90171b6a836449f8d0e982d5d71e5c5a
+        uses: axel-op/googlejavaformat-action@v3
```

Comment:
Thanks for catching this! Could you please revert this line because the GitHub action releases need be tagged with hashes per our repo's security policy, and otherwise dependabot may overwrite this line later.

Reply:
I see, I was wondering why we use hash, thanks. The current hash is quite old, `skip-commit: true` fails. I'll take this part out to a separate patch.

Reply:
Sounds good to me. Thank you!

Reply:
I made a follow-up PR at https://github.com/google/budoux/pull/363